tools/tsp-client/src/typespec.ts

```diff
@@ -100,7 +100,8 @@ export async function compileTsp({
   Logger.debug(`Compiler options: ${JSON.stringify(options)}`);
   if (diagnostics.length > 0) {
     // This should not happen, but if it does, we should log it.
-    Logger.debug(`Compiler options diagnostic information: ${JSON.stringify(diagnostics)}`);
+    Logger.error(`Compiler options diagnostic information: ${JSON.stringify(diagnostics)}`);
```

Comment:
@timotheeguerin can we also use `formatDiagnostic()` with the diagnostics returned from the `resolveCompilerOptions()`?

Reply:
Also, do you have an example of a configuration that would cause an error? I'd like to test this out.

Reply:
Yep, any typespec diagnostic